Userland/Libraries/LibWeb/HTML/HTMLMediaElement.cpp

```diff
@@ -48,10 +89,679 @@ void HTMLMediaElement::load() const
     dbgln("(STUBBED) HTMLMediaElement::load()");
```

Comment:
Quick and easy addition :^):
```suggestion
    // When the load() method on a media element is invoked, the user agent must run the media element load algorithm.
    load_element();
```

Reply:
yeahhh I know that one's super easy, but I was trying not to add a bunch more here since the `load_element` commit is already so big. Planning on banging out load/play/pause/etc in a short follow-up PR.

(for reference, `load` does not help youtube get any further :upside_down_face: )